src/program/config/data_format/data_format.lua

```diff
@@ -0,0 +1,135 @@
+-- Use of this source code is governed by the Apache 2.0 license; see COPYING.
+module(..., package.seeall)
+
+local lib = require("core.lib")
+local schema = require("lib.yang.schema")
+local yang_data = require("lib.yang.data")
+local usage = require("program.config.data_format.README_inc")
+
+-- Number of spaces a tab should consist of when indenting config.
+local tab_spaces = 4
```

Comment:
We should use the same as what the serializer produces, which is currently 2.  Can you use 2 here?